### DISCUSSION TITLE
[Feature Request] Make keymap different based on the application name instead of class

### DISCUSSION THREAD

**Discussion by zw963** [xremap/xremap#374] [Ideas]
Following output of `wmctrl` is a example.

the app in following example is a compare software, as you can see, based on different page(Text Compare/Folder Compare) in this app,
setting different keybinding is reasonable.

```
$: wmctrl -lx
0x03c00060  0 bcompare4.Bcompare4   lggram bottom-update - Text Compare - Beyond Compare
```

```cl
$: wmctrl -lx
0x03c00060  0 bcompare4.Bcompare4   lggram bin <--> sbin - Folder Compare - Beyond Compare
```

So, Can we different the `Text Compare - Beyond Compare` with `Folder Compare - Beyond Compare` use same rule as window class?

thank you.

**Comment by k0kubun** [comment]
We're tracking this at https://github.com/k0kubun/xremap/issues/150.
